src/pages/NewChatPage.js

```diff
@@ -175,7 +184,7 @@ class NewChatPage extends Component {
                 this.props.reports,
                 this.props.personalDetails,
                 this.props.betas,
-                isOptionInList ? prevState.searchValue : '',
+                searchValue,
```

Comment:
```suggestion
                isOptionInList ? prevState.searchValue : ''
```

Reply:
We would have to check for `this.props.isGroupChat` too, if `isOptionInList` is `false`, we will get the same behaviour, yo were encountering [here](https://github.com/Expensify/App/pull/7530#issuecomment-1053598677)

Is `(isOptionInList || this.props.isGroupChat) ? prevState.searchValue : ''` acceptable?

Reply:
`toggleOptions` only runs when the page is groupChat.

Reply:
> `toggleOptions` only runs when the page is groupChat.


Got it.
Then we just need to pass `prevState.searchValue`, since `isOptionInList` is `true` only when you **de**select an option from the list